pytests/test_transforms.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_almost_equal
+
+from pylops.transforms import (
+    DiscreetCosine
+)
+from pylops.utils import dottest
+
+par1 = {"ny": 11, "nx": 11, "imag": 0, "dtype": "float64"}
+par2 = {"ny": 11, "nx": 21, "imag": 0, "dtype": "float64"}
+par3 = {"ny": 21, "nx": 21, "imag": 0, "dtype": "float64"}
+
+@pytest.mark.parametrize("par", [(par1), (par3)])
+def test_DiscreetCosine1D(par):
+    """Dot test  for Discreet Cosine Transform Operator 1D"""
+
+    t = np.arange(par["ny"])
+
+    Dct = DiscreetCosine(dims=(par["ny"],), dtype=par["dtype"])
+
+    assert dottest(Dct, par["ny"], par["ny"], rtol=1e-6, complexflag=0, verb=True)
+    
+    y=Dct.H*(Dct*t)
```

Comment:
This needs to be run through flake8 (with our pre-commit) as lots of spaces are missing

Reply:
yes, I used pre-commit this time

Reply:
Great :)